TalkAloud/TalkAloud/Managers/AudioManager.swift

```diff
@@ -18,11 +18,19 @@ class AudioManager {
     private var audioRecordings: [AudioRecording] = []
     private var filteredAudioRecordings: [AudioRecording] = []
     private var allUniqueTags: [String] = []
-
+    
     private var didNewRecording = false
     
     private init() {}
     
+    // MARK: - Public Methods
+    
+    func loadAudioRecordings(with tags: [String]?) -> [AudioRecording] {
```

Comment:
Make this optional because it will crash if you don't have any elements

Reply:
Should I also make the audioRecrodings property in my AudioRecordingsViewController an optional so I can have a nil value anytime there are no recordings in the app?

Reply:
No. You can keep that array as empty if you don't have smth.

Reply:
Done.